docs/user/intro.md

```diff
@@ -1,80 +1,95 @@
+If you don't have the basics down, please read some of [the main documentation page](README.md) and/or watch [the Cursorless tutorial videos](https://www.youtube.com/watch?v=5mAzHGM2M0k&list=PLXv2sppxeoQZz49evjy4T0QJRIgc_JPqs).
 
-## Some Important Concepts
+It is planned for this guide to be expanded to both basic and advanced levels.
 
-### Commands Have An Action And A Target Or Two
+## Some important concepts
+
+### Commands have an action and a target or two
 
 Commands have an action and a target or two. `"chuck air"` has the action `"chuck"` and one target (the `"air"` token).
 `"bring air to bat"` has the action `"bring"` and two targets (the `"air"` and `"bat"` tokens).
 
-A target can be more than a single token or line. A target can be a range, like `"air past bat"` that spans from the `"air"` token to the `"bat"` token, and that is still considered _one_ target.
+A target can be more than a single token or line. A target can be a [range](README.md#range-targets), like `"air past bat"` that spans from the `"air"` token to the `"bat"` token, and that is still considered _one_ target.
 A target can even be a vertical slice or list, like the list `"air and bat"`.
 Doing `"take air and bat"` will result in multiple selections and multiple cursors, but it is still one target.
 
-Commands with two targets often require a separator word between the two targets, like the `"to"` in `"move air to bat"`.
-The separator may seem trivial now, but it is easier to forget it when you start using fancy targets.
-
-### Targets Can Be Modified
+### Targets can be modified
 
-A target starts as a mark (`"this"`, `"row ten"`, `"green curve air"`, etc) and target modifiers can be applied to suit your needs.
+A target contains a mark (ex: `"this"`, `"row ten"`, `"green curve air"`, etc) and target modifiers can be prefixed to suit your needs.
 A big part of feeling comfortable and productive with Cursorless is getting familiar with the most useful mark types and target modifiers.
 All of the [mark types](README.md#Marks) are worth knowing.
 While there are many target modifiers, this document will help you get started fast by focusing on the most important modifiers to learn first.
 
-The big difference between a simple command `"bring air to bat"` and the intimidating `"bring next three tokens air to after just line bat"` is the addition of target modifiers.
-Target modifiers are often chained. `"after just line"` is a chain of three modifiers and they are applied in reverse order (`"line"`, then `"just"`, then `"after"`).
+The biggest difference between the simple `"bring air to bat"` and the intimidating `"bring next three lines air to tail block bat"` is the addition of target modifiers.
+Target modifiers are often chained. `"tail block"` is a chain of two modifiers and they are applied in reverse order (`"block"`, then `"tail"`).
```

Comment:
`"tail block"` is actually a single modifier 😅.  It extends its input past the end of its containing block

Reply:
If you're looking for an intimidating command, maybe grab one from one of my videos?  eg https://youtu.be/xtOkYdwUves (`"move line every instance square plex after file"`).  Tho keep in mind in that example the second target (the destination `"after file"`) doesn't actually have an explicit mark; it's using the implicit `"just this"` mark

But the source does break down as a nice long chain.  Reading from right to left, it's:

1. plex
2. square ("box" in default parlance)
3. every instance
4. line

Reply:
I was hoping to have an example that would be fully explained by the current document; "every instance" is going to be in the next document.  I can just change to "block func".